SwiftUIDemoAppUITests/Tests/DeeplinkTests.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  DeeplinkTests.swift
+//  SwiftUIDemoAppUITests
+//
+//  Created by Ilias Pavlidakis on 25/9/23.
+//
+
+import XCTest
+
+final class DeeplinkTests: StreamTestCase {
+
+    static override func setUp() {
+        super.setUp()
+
+        // We are launching and terminating the app to ensure the executable
+        // has been installed.
+        app.launch()
+        app.terminate()
+    }
+
+    override func setUpWithError() throws {
+        launchApp = false
+        try super.setUpWithError()
+    }
```

Comment:
Why do we need `launchApp = false` if we then anyways launch and terminate the app? The app should be installed without launching and terminating the app 🤔

Reply:
We are only launching the app on the static setUp which is being called only once (just before any test case from the suite start running). In that way, we don't take the hit of that step every time but only once. 

> The app should be installed without launching and terminating the app
Amazingly it's not :(